elements/pfe-primary-detail/pfe-primary-detail.ts

```diff
@@ -477,6 +476,11 @@ export class PfePrimaryDetail extends LitElement {
       }));
     }
 
+    if (this.breakpoint === 'compact' && this.active)
+      this._detailsBackButton?.setAttribute('aria-expanded', 'true');
```

Comment:
imperative DOM manipulations on elements that lit-html manages tend to break things. Are you able to refactor this to compute the value in the `render` method?

Reply:
Should be something like
```ts
render() {
  const detailsBackButtonExpanded = 
      this.breakpoint === 'compact' && this.active ? 'true'
    : undefined
  return html`
    ...
    <button
        id="details-wrapper__back
        aria-expanded="${ifDefined(detailsBackButtonExpanded)}">
  `;
}

Reply:
@wesruv I made this change just now (see force-push, below). I also added a test in the spec file. PTAL